src/Core/src/Core/IWebView.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Represents a View that presents HTML content.
+	/// </summary>
+	public interface IWebView : IView
+	{
+		/// <summary>
+		/// Provide the data for a WebView.
+		/// </summary>
+		IWebViewSource Source { get; set; }
```

Comment:
`WebViewSource` in Xamarin.Forms uses BindableProperties to notify and update changes. What should be the strategy to follow in this case? A common interface and different types?